### ISSUE TITLE
TextInput inside of a Box loses focus/state

### ISSUE THREAD

**Issue by nickharder88** [open-olive/loop-development-kit#329]
Reproduction steps - Put a TextInput inside of a Box and try to type in it.

**Comment by nickharder88** [comment]
This occurs whether it's a controlled or uncontrolled component

**Comment by BrettMugglinOlive** [comment]
Good catch. We've reproduced the bug that you found, and opened a ticket to address it. Should be able to make it in for v3.3.0.

**Comment by richseviora** [comment]
This bug's fix has been merged into our main branch and will be fixed in the next release.

Internal ticket - HELPS-1381
